### PR TITLE
changed DateRangePicker

### PR DESCRIPTION
**Pull request by mvmoshchovskyi**
changed DateRangePicker

1-instelled new library rsuite
2-new NavFilterByDate
3-changed use_history-filters
3-changed use_orde_-filters

dateRangePicker

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
-        </form>
-    );
+  const [value, setValue] = useState([from, to]);
+  const setDateHandler = (e) => {
+    setValue(e);
+    dateHandler(e);
+  };
+  return (
+    <DateRangePicker
+      format='DD-MM-YYYY'
```
Comment (AndriyLymych) [inline src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js:21]: take format from consts

File: src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js [hunk 2]
```diff
@@ -1,48 +1,42 @@
...
+      isoWeek
+      style={{ width: 280 }}
```
Comment (AndriyLymych) [inline src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js:23]: don't use inline styles

File: src/components/filter-search-sort/filter-navbar.js [hunk 3]
```diff
@@ -1,66 +1,73 @@
...
-                    <NavClearFilters clearOptions={clearOptions}/>
-                </Grid>
-            ) : null}
+  return (
+    <Grid className={styles.container}>
+      {/* {filterByDateOptions?.length ? <div className={styles.dateRange}>{ */}
+      {/*    filterByDateOptions.map(filterItem => (<Grid className={styles.dateRangeItem} key={filterItem} item> */}
+      {/*        <NavFilterByDate filterByDateOptions={filterItem}/> */}
+      {/*    </Grid>)) */}
+      {/* }</div> : null} */}
```
Comment (AndriyLymych) [inline src/components/filter-search-sort/filter-navbar.js:27]: remove useless code

File: src/components/filter-search-sort/filter-navbar.js [hunk 4]
```diff
@@ -1,66 +1,73 @@
...
 
 FilterNavbar.defaultProps = {
-    options: {},
-    sortOptions: {},
-    filterByDateOptions: [],
-    searchOptions: {},
-    clearOptions: {}
+  options: {},
+  sortOptions: {},
+  //   filterByDateOptions: [],
```
Comment (AndriyLymych) [inline src/components/filter-search-sort/filter-navbar.js:69]: same

**Review by AndriyLymych — CHANGES_REQUESTED** [review]
please, also add this filter for comment

**Comment by AndriyLymych** [comment]
image
please, set styles like in whole admin part

File: src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js (hunk 1 above)
Comment (mvmoshchovskyi) [reply src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js]: fixed

File: src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js (hunk 2 above)
Comment (mvmoshchovskyi) [reply src/components/filter-search-sort/filter-by-date/nav-filter-by-date.js]: fixed

File: src/components/filter-search-sort/filter-navbar.js (hunk 3 above)
Comment (mvmoshchovskyi) [reply src/components/filter-search-sort/filter-navbar.js]: fixed

File: src/components/filter-search-sort/filter-navbar.js (hunk 4 above)
Comment (mvmoshchovskyi) [reply src/components/filter-search-sort/filter-navbar.js]: fixed

**Comment by mvmoshchovskyi** [comment]
> image
> please, set styles like in the whole admin part

styledUA
добавив нові стилі

**Pull request closed without merging**